### PR TITLE
Download star list to boot comet

### PR DESCRIPTION
**Pull request by belisarius222**
dual of https://github.com/urbit/arvo/pull/980

This also:
- fixes a bug where we bunted a dead zuse type
- refactors snapshot downloading to use a new `_dawn_get_jam()` helper

I've initialized the file at the hard-coded URL to include just `~marzod`.

### CODE REVIEW

**Comment by belisarius222** [comment]
Huh, travis noticed a C problem that my Mac didn't. Must be a LLVM/GCC issue. I'll fix that

File: vere/dawn.c [hunk 1]
```diff
@@ -365,7 +370,7 @@ u3_dawn_vent(u3_noun seed)
     else if ( c3__pawn == rank ) {
       //  irrelevant, just bunt +point
       //
-      pot = u3v_wish("*point:constitution:ethe");
+      pot = u3v_wish("*point:azimuth");
```
Comment (joemfb) [inline vere/dawn.c]: Good catch!

File: include/vere/vere.h [hunk 2]
```diff
@@ -1201,12 +1201,12 @@
...
 
-      /* u3_dawn_come(): mine a comet under star (unit)
+      /* u3_dawn_come(): mine a comet
       */
         u3_noun
-        u3_dawn_come(u3_noun star);
+        u3_dawn_come(void);
 
-      /* u3_dawn_vent(): validatated boot event
+      /* u3_dawn_vent(): validated boot event
```
Comment (joemfb) [inline include/vere/vere.h]: Oof

Comment (belisarius222) [reply include/vere/vere.h]: I mean, validatated is obviously a stronger acceptance criterion, but this is still relatively new code.

**Pull request merged**